tools/harness-thci/OpenThread.py

```diff
@@ -1065,14 +1077,11 @@ def joinNetwork(self, eRoleId):
             elif eRoleId == Thread_Device_Role.REED:
                 print('join as REED')
                 mode = 'rsdn'
-                # set ROUTER_UPGRADE_THRESHOLD
-                self.__setRouterUpgradeThreshold(0)
```

Comment:
what are you going to do with a `REED` ?

Reply:
by mistake